open-science-projects.md

```diff
@@ -23,34 +23,58 @@ Discussions on errors, new features, and other specific issues for riboviz, take
+## Projects available to students can be more or less open
 
-The contact hours of an student project depend heavily on the type of project – with a lab-based project requiring lots of contact hours, with students being in the lab with their supervisor. Alternatively for a literature-based research project, or a community outreach project, there would likely be fewer contact hours, as the student would require more guidance rather than constant supervision.
+Students on our undergraduate Biotechnology Honours course had access to a wide variety of projects.
+Potential projects had varying levels of openness.
+A traditional project might involve designing and attempting an experiment (in the absence of a pandemic), or a computational analysis, or scouring literature with the goal of answering a research question. 
+The immediate outputs of the project would usually be talks to the hosting lab and the degree program, the written report, and any results and reagents generated by the student.
+Any impact beyond the hosting lab, for example contributions leading to a publication, might not be visible until long after the student has left.
+
+Our project involved jumping into a team working on the riboviz data analysis pipeline.
+We contributed to the overall development of riboviz in the form of adding new datasets, testing the pipeline, and commenting on its usability.
+Our work is documented on GitHub and is publicly accessible, providing proof of our contributions to the overarching project, in addition to our own reports.
+Sharing our contributions instantly and openly provides concrete evidence of our capabilities, which is accessible to potential employers.
+
+Some other available projects involved working with trade secrets or compounds with therapeutic potential, with appropriate legal restrictions on confidentiality. 
+The idea of navigating an unfamiliar set of rules for protecting intellectual property caused us a lot of anxiety, which we avoided by working on an open science project.
+
+The contact hours of an student project depend heavily on the type of project.
+A lab-based project requires lots of contact hours in the lab with the day-to-day supervisor.
+Alternatively for a literature-based research project, or a community outreach project, there would likely be fewer contact hours, as the student would be less likely to set anything on fire. EDWARD ASKS: IS THIS FUNNY? WORTH INCLUDING? (it is true, yes, lab students do sometimes set things on fire.)
```

Comment:
re: 
> as the student would be less likely to set anything on fire. 
> EDWARD ASKS: IS THIS FUNNY? WORTH INCLUDING? (it is true, yes, lab students do sometimes set things on fire.) 

I'm not sure this is worth including, I don't know if it fits the rest of the tone/content?

Reply:
We agreed that it’s not worth including. Better to cut down to start with the next part about contact hours for riboviz only.